test/jmh/BUILD

```diff
@@ -1,6 +1,27 @@
-load("//jmh:jmh.bzl", "scala_benchmark_jmh")
+load("//jmh:jmh.bzl", "scala_library", "scala_benchmark_jmh")
+
+native.java_library(
```

Comment:
I didn't even know you could write `native.java_library` but `java_library` should work.